CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to HTTPS Everywhere
+
+If you want to create new rules to submit to us, we expect them to be in the src/chrome/content/rules directory. That directory also contains a useful script, make-trivial-rule, to create a simple rule for a specified domain. There is also a script called trivial-validate.py, to check all the pending rules for several common errors and oversights. For example, if you wanted to make a rule for the example.com domain, you could run:
```

Comment:
I would note that `trivial-validate.py` is located within the `utils/` path just to be clear.